### PR TITLE
Automated install running screen again

### PR DESCRIPTION
**Pull request by cbaenziger**
This adds starting screen in `automated_install.sh` again. We launch to `wait-for-hosts.sh` but still allow one to easily catch the VM's booting without having to race the VM's. If one does not have `screen` installed it falls back to only `wait-for-hosts.sh`. I needed to get to the VM's quickly to see the step in the installer on which they failed -- screen launching at boot helped me a lot with this.

### CODE REVIEW

**Comment by cbaenziger** [comment]
Oof; forgot to add the `kill screen` back in to make sure it exits out happily once all VM's start...

File: tests/automated_install.sh [hunk 1]
```diff
@@ -97,23 +100,28 @@ if hash screen; then
   fi
 
   # Create a new screenrc with our VM list in it
+  SCREENRC=../../cluster/screenrc
   cp ./screenrc ../../cluster/screenrc
```
Comment (vt0r) [inline tests/automated_install.sh]: Probably want to use that shiny new variable right here :)

File: tests/automated_install.sh [hunk 2]
```diff
@@ -97,23 +100,28 @@ if hash screen; then
   fi
 
   # Create a new screenrc with our VM list in it
+  SCREENRC=../../cluster/screenrc
```
Comment (vt0r) [inline tests/automated_install.sh]: Can we use an absolute path here?

File: tests/automated_install.sh [hunk 3]
```diff
@@ -97,23 +100,28 @@ if hash screen; then
...
+         "./tests/virtualbox_serial_console.sh $vm" >> $SCREENRC
     ((ii++))
   done
+  echo "screen -t \"Wait For Hosts\" 0 $WAIT_FOR_HOST_CMD; $KILL_SCREEN_CMD" >> $SCREENRC
   popd > /dev/null
-  echo "Enter this command to view VM serial consoles:"
-  echo "  screen -S 'BACH serial consoles' `readlink -f ../cluster/screenrc`"
-  echo
+
+  screen -S 'BACH VM Install Progress' -c `readlink -f ../cluster/screenrc`
```
Comment (vt0r) [inline tests/automated_install.sh]: Also probably want to use the variable here, assuming an absolute path.

**Comment by cbaenziger** [comment]
In #1242 serial support for VM clusters was slid in; closing this as needless now.

**Pull request closed without merging**